### PR TITLE
Fix for ButtonKit & Signals Docs

### PR DESCRIPTION
**Pull request by m1-dev**
I've changed two files.

- ButtonKit Documentation: I made comments start with capital letters.
- Signals Documentation: In the previous documentation, both CommonJS, ESM, and TypeScript examples used TypeScript code (e.g. ActionRowBuilder<ButtonKit> in the JS docs), so I fixed that. I've also changed comments for more consistency.

### CODE REVIEW

**Review by notunderctrl — CHANGES_REQUESTED** [review]
[`using-signals.mdx`](apps/docs/pages/docs/using-signals.mdx): Instead of variable names such as `disposeValue` and `disposeCount`, I think names such as `disposeValueSubscribers` and `disposeCountSubscribers` would be more descriptive.

File: apps/docs/pages/docs/using-signals.mdx [hunk 1]
```diff
@@ -18,45 +18,60 @@ Creating a signal is simple, you just need to call the `createSignal` function f
...
+        // Dispose subscribers
+        disposeValue();
         ```
     </Tabs.Tab>
     <Tabs.Tab>
         ```js filename="signal.js" copy
         import { createSignal } from 'commandkit';
 
-        const [value, setValue, dispose] = createSignal(0);
+        // createValue accepts one param (the initial value)
```
Comment (twlite) [inline apps/docs/pages/docs/using-signals.mdx:41]:
```suggestion
        // createSignal accepts one param (the initial value)
```

Shouldn't this be `createSignal`

**Review by twlite — CHANGES_REQUESTED** [review]

**Comment by m1-dev** [comment]
> For `using-signals.mdx`: Instead of variable names such as `disposeValue` and `disposeCount`, I think names such as `disposeValueSubscribers` and `disposeCountSubscribers` would be more descriptive.

Okay, I'll rename them.

**Comment by m1-dev** [comment]
@notunderctrl I've renamed the variables to use the better names you've suggested. A review would be appreciated.

**Pull request merged**